portal/templates/contract/components/abandon-deal-link.njk

```diff
@@ -9,8 +9,9 @@
           data-cy="AbandonLink"
           href="/contract/{{ params.deal._id }}/delete"
           class="govuk-link govuk-!-font-size-14 govuk-!-margin-left-2"
-          aria-label="Abandon Deal"
-        >Abandon</a>
+          title="Abandon Deal {{ params.deal.bankInternalRefName }}"
+          aria-label="Abandon Deal {{ params.deal.bankInternalRefName }}"
+        >Abandon {{ params.deal.bankInternalRefName }}</a>
```

Comment:
how does it render on the page if the bankInternalRefName is very long?

Reply:
and is this applicable to both GEF and BSS deals?

Reply:
sent what it looks like on slack.  And currently only on BSS.  Not got the tickets for GEF analysis yet

Reply:
done